var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from shutil import copytree, copy
```

Comment:
you can use spack's `install_tree` and `install` for that :)

Reply:
Thanks for the suggestion.  I've made the change to using spack's install_tree().